python3/cloudwatch/src/lambda_function.py

```diff
@@ -136,6 +145,31 @@ def _is_valid_log(log):
         'END') or message.startswith('REPORT')
     return not is_info_log
 
+def _has_whitelist():
+    # type: () -> bool
+    return os.environ.get('WHITELIST') is not None
+
+def _is_whitelisted_log(log):
+    # type: (dict) -> bool
+    sep = os.environ.get('WHITELIST_SEPARATOR', ';')
```

Comment:
Please have the separator non-configurable (`;`).

Reply:
I'd prefer to keep it configurable. Sometimes you want to blacklist log strings where there is a semicolon present. Configurable separator allows you to use something like `,` or `|` .